cohere/response.py

```diff
@@ -1,5 +1,5 @@
 class CohereObject():
-    def __str__(self) -> str:
+    def __repr__(self) -> str:
```

Comment:
Should we add a test for this?

Reply:
Good call. I added a test for classify. Adding one for embed and generate is trickier because they are not deterministic

Reply:
I think just the one is fine.